Amazon.IonHashDotnet/IonHashReader.cs

```diff
@@ -223,14 +223,6 @@ public int GetLobByteSize()
             return this.reader.GetLobByteSize();
         }
 
-        /// <summary>
```

Comment:
Duplicate Dispose() function that was added by me accidentally in previous PR because I didn't rebase.